src/video/resources/SigningKeys.js

```diff
@@ -0,0 +1,106 @@
+/*!
+ * Mux Assets
```

Comment:
`Mux SigningKeys`

Reply:
Derp, missed this when I pulled over the base for all the comments and such. Yeah we might just be able to put one of these in the base or at the LICENSE level...we can chat after the holidays about that one.